data/commodities.txt

```diff
@@ -709,3 +709,126 @@ trade
 		"stolen yachts"
 		"undeclared nuclear fuel"
 		"weaponized chickens"
+	commodity "Library Media"
+		"hardback books"
+		"webcomic printouts"
+		"softcover books"
+		"science fiction novels"
+		"cheesy romance novels"
+		`deathcore "song" recordings`
```

Comment:
The quotes seem like unnecessary editorializing to me :)

Reply:
As a Deathcore fan who doesn't like music in general, I find your statement deeply offensive. It is an insult to Deathcore to call its fine auditory masterpieces mere songs. They are genuine statements of our primal inner selves, expressed in a way that no other form of art has ever even approached. It is music taken to the extreme limit of human tolerance, so far that it could even be used as a weapon. It is like the California Reaper of music; at the far end of human tolerance, and only for those with the strength of character to handle it.

The fact that Deathcore has been known to cause seizures and migraines is completely irrelevant.

Reply:
I have changed it to "deathcore masterpieces" which is more appropriate given the context.